src/schema/ecommerce/create_order_mutation.js

```diff
@@ -0,0 +1,110 @@
+import {
+  GraphQLInputObjectType,
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLList,
+  GraphQLInt,
+  graphql,
+} from "graphql"
+import { OrderType } from "schema/ecommerce/types/order"
+import { mutationWithClientMutationId } from "graphql-relay"
+
+const LineItemInputType = new GraphQLInputObjectType({
```

Comment:
im not too familiar with file structure here either, what do you think about moving input types to their own folder and have this file only define the mutation by importing these input types?

Reply:
I thought about it and decided to move InputTypes to the Types folder when they are reused in other queries/mutations. I will move them as I add more mutations.